docs/source/user_guide/gpu.rst

```diff
@@ -45,17 +50,255 @@ For example, on a Thinkpad AMD laptop with a dedicated nvidia GPU this returns::
     llvmpipe (LLVM 15.0.6, 256 bits) (CPU) on Vulkan
     AMD Radeon Graphics (rembrandt, LLVM 15.0.6, DRM 3.52, 6.4.0-0.deb12.2-amd64) (Unknown) on OpenGL
 
+In jupyter all the available adapters are also listed when ``fastplotlib`` is imported.
+
+You can get more detailed info on each adapter like this::
+
+    print([a.request_adapter_info() for a in fpl.enumerate_adapters()])
```

Comment:
Similarly, the list of dicts being printed is not easy to read. Using json or pprint is much friendlier:
```suggestion
    import pprint

    for a in fpl.enumerate_adapters():
        pprint.pprint(a.request_adapter_info())
```